src/wallet/test/ismine_tests.cpp

```diff
@@ -143,7 +224,16 @@ BOOST_AUTO_TEST_CASE(ismine_standard)
         BOOST_CHECK_EQUAL(result, ISMINE_NO);
     }
 
-    // (P2PKH inside) P2SH inside P2WSH (invalid)
+    // (P2PKH inside) P2SH inside P2SH (invalid) - Descriptor
+    {
+        CWallet keystore(chain.get(), "", m_args, CreateDummyWalletDatabase());
+        std::string desc_str = "sh(sh(" + EncodeSecret(keys[0]) + "))";
+
+        auto spk_manager = CreateDescriptor(keystore, desc_str, false);
+        BOOST_CHECK_EQUAL(spk_manager, nullptr);
+    }
```

Comment:
Is `ismine_tests.cpp` the right place for this test? It doesn't seem to be testing `IsMine`. There are a few other places where it seems like `IsMine` isn't being tested, namely wherever the script is invalid and `CreateDescriptor` returns a nullptr.

Reply:
Right. This happens in the cases below:

* P2PKH inside) P2SH inside P2SH (invalid)
* (P2PKH inside) P2SH inside P2WSH (invalid)
* P2WPKH inside P2WSH (invalid)
* (P2PKH inside) P2WSH inside P2WSH (invalid)
* P2WPKH uncompressed (invalid)
* P2WSH multisig with uncompressed key (invalid)

These cases cannot be reproduced for descriptor ScriptPubKeyMan, so I added them just for consistency and to make this difference explicit.
I can remove them if the reviewers think this is not necessary.